tests/TestHelpers/ObjectSerialization/JsonSerializable.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Rollbar\TestHelpers\ObjectSerialization;
```

Comment:
Nit: these may not pass linting, as we have `namespace` beside the `<?php` in other files. However, we will be changing this in the next major iteration of this library in order to support `strict_types`.